flink-filesystems/flink-s3-fs-hadoop/src/main/resources/META-INF/NOTICE

```diff
@@ -30,7 +30,7 @@ This project bundles the following dependencies under the Apache Software Licens
 - org.apache.hadoop:hadoop-common:3.1.0
 - org.apache.htrace:htrace-core4:4.1.0-incubating
 - org.apache.httpcomponents:httpcore:4.4.6
```

Comment:
why are we not bumping httpcore as well?

Reply:
I didn't want to create a PR with multiple dependencies. If you prefer, I can definitely include it. Let me know.

Reply:
In practice it is part of httpclient so I would prefer to bump them at once.